zap/src/main/java/org/zaproxy/zap/control/CoreFunctionality.java

```diff
@@ -142,24 +141,7 @@ private static synchronized void createExtensions() {
      * @see AbstractPlugin
      */
     public static List<AbstractPlugin> getBuiltInActiveScanRules() {
```

Comment:
Remove it?

Reply:
Kept it because it was public, but I don't mind removing it.

Reply:
I think we can consider this class internal, it's to bootstrap ZAP. We can also deprecate it first.